crates/as/index.ts

```diff
@@ -218,7 +218,7 @@ function headersToString(headers: Map<string, string>): string {
     let keys = headers.keys() as Array<string>;
     let values = headers.values() as Array<string>;
     for (let index = 0; index < keys.length; ++index) {
-        res += '"' + keys[index] + '"' + ":" + '"' + values[index] + '"\n';
```

Comment:
This worked fine when deserializing JSON, but the additional quotation marks when passing plain text meant the headers weren't being sent properly.